### PR TITLE
Test

### PR DESCRIPTION
**Pull request by johspaeth**

### CODE REVIEW

File: src/main/java/test/A.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package test;
+
+public class A {
+  Object o;
+  public static void main(String...args) {
+    A a = new A();
+    a.o.toString();
+  }
```
Comment (mbenz89) [inline src/main/java/test/A.java]: Potential **null** derefence

File: src/main/java/test/A.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package test;
+
+public class A {
+  Object o;
+  public static void main(String...args) {
+    A a = new A();
+    a.o.toString();
```
Comment (mbenz89) [inline src/main/java/test/A.java]: Potential **null** derefence

Comment (johspaeth) [reply src/main/java/test/A.java]: Do you mean from that line?
https://github.com/johspaeth/PathExpression/blob/199af867c7687a328b1307e19485a3914188cc0d/src/main/java/test/A.java#L4

Comment (mbenz89) [reply src/main/java/test/A.java]: https://github.com/johspaeth/PathExpression/blob/199af867c7687a328b1307e19485a3914188cc0d/src/main/java/test/A.java#L5

**Pull request closed without merging**